Build/Projects/MonoGame.Framework.definition

```diff
@@ -1190,6 +1190,9 @@
     <Compile Include="Utilities\ReflectionHelpers.Legacy.cs" >
       <Platforms>Angle,Linux,MacOS,Windows,WindowsGL,Web</Platforms>
     </Compile>
+    <Compile Include="Utilities\TimerHelper.cs">
+      <Platforms>Windows,WindowsGL</Platforms>
```

Comment:
It's only WindowsDX that needs this. WindowsGL does not define the WINDOWS symbol so it won't use the Timer helper.

Reply:
Are you sure? The WindowsGL project definitely has this symbol defined for me:
![image](https://user-images.githubusercontent.com/5249498/48405844-0d475e80-e76e-11e8-90c8-d9810ed0bbc5.png)
Also from the Protobuild `MonoGame.Framework.definition`:
```
    <CustomDefinitions>
      ...
      <Platform Name="WindowsGL">TRACE;WINDOWS;DESKTOPGL;SUPPORTS_EFX</Platform>
      ...
    </CustomDefinitions>
```
Do you still want me to remove this?

Reply:
Sorry, you're right. I didn't check. I thought it wasn't there in the Protobuild definition file because we want DesktopGL to have cross platform builds (though Mac uses some platform specific stuff already). It's fine as is for now :) I'll see about removing the symbol definition in a later PR, because there might be some complications.